MeshLib/MeshInformation.cpp

```diff
@@ -63,4 +65,62 @@ std::array<unsigned, 7> MeshInformation::getNumberOfElementTypes(
     return n_element_types;
 }
 
+void MeshInformation::writeAllNumbersOfElementTypes(const MeshLib::Mesh& mesh)
+{
+    auto const& nr_ele_types =
+        MeshLib::MeshInformation::getNumberOfElementTypes(mesh);
+    const std::array<std::string, 7> element_names{
```

Comment:
This mapping is already implemented in MeshLib/MeshEnums.h and the corresponding mapping can by achieved as follows:

```c++
for (int element_type = 0; element_type < nr_ele_types.size();
         element_type++)
    {
        INFO("\t%d %s", nr_ele_types[element_type],
             MeshLib::MeshElemType2String(
                 static_cast<MeshLib::MeshElemType>(element_type)).c_str());
    }
```

Reply:
Well, I tried it, and the output was just 'none' so I rewrote this mapping.

Reply:
I think the remaining question is, whether I should add a new mapping to MeshEnums or I should apply the VTK mapping to the array (meaning that we are wasting memory) or whether I should keep it as it is.

Reply:
I don't understand why the element type to element name mapping is inconsistent. In what cases 'none' is returned? Which cases are handled wrong? Could you please provide a test example?

Reply:
Just discussed with @joergbuchwald. The OGS-enum is inconsistent with the array provided by the method at hand. It is @joergbuchwald's decision whether to change the array or not.

Reply:
I think I stick with it for the moment. 
When I will have some time, I will change it to the VTK mapping as defined in MeshEnums.h but in a separate PR.